elixir/apps/web/lib/web/live/gateways_live/show.ex

```diff
@@ -162,52 +193,19 @@ defmodule Web.GatewaysLive.Show do
       </div>
     </div>
     <div class="relative overflow-x-auto">
-      <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
-        <thead class="text-xs text-gray-900 uppercase dark:text-gray-400">
-          <tr>
-            <th scope="col" class="px-6 py-3">
-              Name
-            </th>
-            <th scope="col" class="px-6 py-3">
-              Address
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr class="bg-white dark:bg-gray-800">
-            <th
-              scope="row"
-              class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white"
-            >
-              <.link
-                navigate={~p"/#{@subject.account}/resources/DF43E951-7DFB-4921-8F7F-BF0F8D31FA89"}
-                class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
-              >
-                Engineering GitLab
-              </.link>
-            </th>
-            <td class="px-6 py-4">
-              gitlab.company.com
-            </td>
-          </tr>
-          <tr class="border-b dark:border-gray-700">
-            <th
-              scope="row"
-              class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white"
-            >
-              <.link
-                navigate={~p"/#{@subject.account}/resources/DF43E951-7DFB-4921-8F7F-BF0F8D31FA89"}
-                class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
-              >
-                SJC VPC-1
-              </.link>
-            </th>
-            <td class="px-6 py-4">
-              172.16.45.0/24
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <.table id="resources" rows={@resources} subject={@subject}>
```

Comment:
I think you don't need to assign it here:
```suggestion
      <.table id="resources" rows={@resources}>
```

Reply:
Similar to one of the comments above, right now the `subject` is needed to get the `account_id` for all of the `navigate` URLs in the links in the table.  If there is a better way of doing this, I'm definitely happy to make the change.

Reply:
@bmanifold for such cases you can assign `subject.account` instead of passing the full struct. This will also have another benefit other than exposing potentially sensitive things to HTML rendering - less state will be passed around.

Reply:
Please check if you can use `navigate={~p"/#{@subject.account}/resources/#{resource.id}"}` without assigning a subject at all too? I don't remember exactly but feel like `@subject` assign will be picked up from the parent context anyways.

Reply:
I think I tried this and it didn't work as I was expecting, so I added it as an attribute.  I will give it another shot though.  I may be misremembering.

If it does not work I'll change the component to take an `account_id` rather than a `subject`

Reply:
Thank you. Sorry I don't remember how internally ~H sigil works with scope of contexts when you use `<.component` :(

Reply:
It looks like your suggestion worked 🙌 

I'm guessing I must have had another issue during the time I was trying this and I had made this change along with another change, then thought adding the subject is what fixed my problem.  All that said, I've removed it from both the template and the component.